modules/olm-removing-catalogs.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * operators/admin/olm-managing-custom-catalogs.adoc
+
+[id="olm-removing-catalogs_{context}"]
+= Removing custom catalogs
+
+As a cluster administrator, you can remove custom Operator catalogs that have been previously added to your cluster by deleting the related catalog source.
+
+.Procedure
+
+. In the *Administrator* perspective of the web console, navigate to *Administration* -> *Cluster Settings*.
+
+. Click the *Global Configuration* tab, then click *OperatorHub*.
+
+. Click the *Sources* tab.
+
+. Select the *Options* menu {kebab} for the catalog you want to remove, and click *Delete CatalogSource*.
```

Comment:
s/for the catalog you want/for the catalog that you want
s/and click/and then click